stl/inc/xstring

```diff
@@ -355,6 +361,9 @@ public:
     using pos_type   = streampos;
     using off_type   = streamoff;
     using state_type = mbstate_t;
+#if _HAS_CXX20
+    using comparison_category = strong_ordering;

```

Comment:
This was removed in my last commit of my previous PR. Not sure if there is a reason to add it again.

Reply:
Also intentional, see https://github.com/microsoft/STL/pull/1678#discussion_r581445639. Thanks for the careful review - commits being lost or damaged somehow are definitely a potential danger when merging so much work so rapidly, so being on the lookout for code mysteriously re-appearing is important! :heart_eyes_cat: